master-nonlatent/master.cfg

```diff
@@ -344,7 +342,7 @@ c['multiMaster'] = True
 
 c['mq'] = {  # Need to enable multimaster aware mq. Wamp is the only option for now.
     'type' : 'wamp',
-    'router_url': 'ws://crossbar:8085/ws',
+    'router_url': os.getenv('MQ_ROUTER_URL', default='ws://localhost:8085/ws'),
```

Comment:
the default should be `crossbar:8085` (since this is what we will use in the future).